api/src/main/resources/org/openmrs/api/db/hibernate/CohortMember.hbm.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+    <class name="CohortMember" table="cohort_member">
+        <id name="cohortMemberId" type="int" column="cohort_member_id"/>
+
+        <property name="cohortId" type="int"
+                 column="cohort_id" not-null="true" length="11"/>
+
+        <property name="patientId" type="int"
```

Comment:
Should be a many-to-one with Patient as the class and no need to specify length for int fields, hibernate can figure out the default